gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/RoleService.groovy

```diff
@@ -0,0 +1,24 @@
+package com.netflix.spinnaker.gate.services;
+
+import com.netflix.spinnaker.gate.services.commands.HystrixFactory
+import com.netflix.spinnaker.gate.services.internal.ClouddriverService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+@Component
+class RoleService {
+  private static final String GROUP = "ecsRoleService"
```

Comment:
Seems strange to have a `RoleService` that's a class, but the impl is ECS specific?